src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -1009,7 +1009,7 @@ bool OutputStateMachineEngine::_GetOscSetColor(const std::wstring_view string,
     }
 
     std::vector<DWORD> newRgbs;
-    for (auto&& part : parts)
+    for (const auto part : parts)

```

Comment:
auto&& should deduce `const auto` here, shouldn't it?

Reply:
It should, and I'm sure it does, but the linter complains (incorrectly), so I just changed it.